### DISCUSSION TITLE
How many epoch does need to train?

### DISCUSSION THREAD

**Discussion by ersanliqiao** [nkicsl/Resfusion#12] [Q&A] [answered]
I think 5000 epoch cost too much time?

**Comment by YukinoshitaLove** [comment, marked as answer]
Yes, 5000 epochs is just a simple setting to ensure convergence. For LOL datasets, about 2000 epochs are enough. For ISTD datasets, about 3000 epochs are sufficient. If you reproduce our experiments, you will find that at the end many epochs only achieved an improvement of about 0.5dB.
